### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by ALIENQuake**
- Alternative Soubar area from the "World of Baldur's Gate" modification - designed and modeled by Cuttooth, with effects enrichment and ideas by Dradiel, and integrated into the mod by Roberciiik.
- Biffing is now an optional installation component.
- Fixed militia training quest when only 9 new guards are trained.
- Fixed CTDs in AR4261, AR4260, and AR4258 areas.
- Restored TIS and MOS files of Soubar for oBG2.
- Fixed a visual bug in Sir William's house (AR1004) in oBG2.
- Updated the worldmap position of areas for Chionthar, Soubar, and Lyrar's Hold.
- Corrected automap notes in Soubar and Soubar prison.
- Traified missing lines (setup, kit name, debug prints).
- Added missing identified description to some letters.
- Moved movies to the language directory so they can be replaced by the local language version.
- Moved some NPC sound files to the language directory - they can now be replaced by the local language version.
- Fixed copying of the MVE folder.
- Uses the %MOD_FOLDER% automatic variable.
- Uses labels for component requirements statements.
- Simplified the mod ID where possible.
- Renamed the mod folder from "SOS" to "SoS" to match the mod name.
- Renamed "setup-SOS.tp2" to "sos.tp2".
- Updated iconv and replaced it with a static build.
- Updated the readme file to use the latest templates.
- Updated the metadata file to use the latest template.
- Removed redundant files.
- Made cosmetic changes to the readme files.

### CODE REVIEW

File: SOS/sos.tp2 [hunk 1]
```diff
@@ -30,65 +30,54 @@ ALWAYS
 	ACTION_IF ~%WEIDU_OS%~ STRING_EQUAL_CASE ~win32~ BEGIN
 		OUTER_SPRINT os_slash ~\~
 		OUTER_SPRINT exe ~.exe~
-		OUTER_SPRINT WEIDU_EXECUTABLE ~setup-SoS.exe~
+		OUTER_SPRINT WEIDU_EXECUTABLE ~setup-%MOD_FOLDER%.exe~
```
Comment (Roberciiik) [inline SOS/sos.tp2]: No need to EVAL? like in 37 line

File: SOS/sos.tp2 [hunk 2]
```diff
@@ -1354,7 +1343,7 @@ BEGIN @21002
 		END
 	END
 
-	COPY_EXISTING ~misc01.itm~ ~override/sos.mrk~
+	COPY_EXISTING ~misc01.itm~ ~override/%MOD_FOLDER%.mrk~
```
Comment (Roberciiik) [inline SOS/sos.tp2]: I would set explicit name here

**Comment by Roberciiik** [comment]
Also there was a report here: http://www.shsforums.net/topic/60697-shadows-over-soubar-v116/?p=612144 about EET text issue. Had a chance to check?

File: SOS/sos.tp2 (hunk 2 above)
Comment (ALIENQuake) [reply SOS/sos.tp2]: [d513e19](https://github.com/SpellholdStudios/Shadows_over_Soubar/pull/27/commits/d513e19430a0b579af110cadfcbd73ea0612865e)

**Comment by ALIENQuake** [comment]
@Roberciiik Sure, I will check.

**Review by Roberciiik — APPROVED** [review]

**Pull request merged**